### PR TITLE
Fix select text color on dark theme

### LINKED ISSUE #138: Text colour of dropdown with dark theme
### Bug Report

- __Package version(s)__: 1.0.1 (?)

#### Describe the issue:

In the docs page http://blueprintjs.com/docs/#components.datetime I noticed that the text colour of drop-down in select is very light, although the background is still white. Here is a screenshot http://prntscr.com/d71f7c I use the latest stable Chrome on Windows 10.

#### Actual behavior:

Light text colour on white background

#### Expected behavior:

Contrasting colours

### PR DESCRIPTION
**Pull request by llorca**
#### PR checklist

- [x] Addresses an existing issue: fixes #138

#### What changes did you make?

OS native `<select>` background color is always white, even inside the dark theme. Therefore the text color should always be `$pt-text-color`, even in the dark theme.

#### Is there anything you'd like reviewers to focus on?

:rotating_light: This hasn't been tested on Windows yet, so we need to make sure the fix works before merging (@cyu06?)

### CODE REVIEW

**Comment by giladgray** [comment]
this must be tested on Windows Chrome (per the original issue). and let's do IE for good measure too.

**Comment by cyu06** [comment]
On it. Will approve review once done.

**Comment by llorca** [comment]
@cyu I moved the `color` and `background-color` to the `<option>` tag, let me know if that works

**Comment by adidahiya** [comment]
Can someone please devtest this on a windows laptop

File: packages/core/src/components/forms/_select.scss [hunk 1]
```diff
@@ -79,7 +80,10 @@ Styleguide components.forms.select
     select {
       @extend %pt-dark-select;
 
-      color: $pt-text-color;
+      option {
```
Comment (giladgray) [inline packages/core/src/components/forms/_select.scss]: this doesn't need to be nested...

**Comment by cmslewis** [comment]
@adidahiya doing now

**Comment by cmslewis** [comment]
Verified that this works in Chrome on Windows:
 
image

image

**Comment by cmslewis** [comment]
Looks good in IE too.

**Pull request merged**